cnd/src/swap_protocols/rfc003/bitcoin/htlc_events.rs

```diff
@@ -61,6 +63,7 @@ impl HtlcEvents<Bitcoin, Amount> for Arc<dyn QueryBitcoin + Send + Sync> {
         Box::new(future::ok(Funded {
             transaction: tx.clone(),
             asset,
+            when: timestamp::now(),
```

Comment:
```suggestion
            when: timestamp::now(),
```
According to the spike, ledger related events will have 2 timestamps:
1. when cnd learns about it
2. the blockchain transaction's timestamp

Knowing that, we should be able to differentiate them via the name, ie, `when` seems too generic to me.
What about `seen`? or `seen_at`?